src/mobile/package.json

```diff
@@ -135,7 +135,8 @@
     "mkdirp": "^0.5.3",
     "**/optimist/minimist": "^0.2.1",
     "minimist": "^1.2.5 || >=0.2.1",
-    "acorn-globals": ">=6.0.0"
+    "acorn-globals": ">=6.0.0",
+    "kind-of": ">=6.0.3"
```

Comment:
```suggestion
    "**/randomatic/kind-of": "^6.0.3",
    "**/sane/micromatch/kind-of": "^6.0.3"
```
I was going to do this but you beat me to it 😄 So the advisory from GitHub is actually incorrect (I emailed GitHub support about this but they haven't replied yet), only versions 6.0.0 through 6.0.2 are vulnerable according to [NPM](https://www.npmjs.com/advisories/1490/versions) and [Snyk](https://snyk.io/vuln/SNYK-JS-KINDOF-537849). Therefore, we only need to make these changes instead of upgrading all instances of `kind-of`.